src/apps/service/apps/job_listings.clj

```diff
@@ -48,7 +48,7 @@
   [{:keys [parent_id id] :as job}]
   (remove-nil-vals
    {:app_description (:app_description job)
-    :app_id          (:app_id job)
+    :app_id          (or (:app_id job) "unknown")
```

Comment:
You may have already, but if you haven't, we should check that anywhere that allows an app ID parameter that flows from this function (or any others changed to allow non-UUID values) won't choke on `"unknown"`. Might be fine because of Agave jobs, but sometimes we have things special-cased, or endpoints that only apply to DE apps, so I think it's worth a review.

Reply:
Yep, it's safe precisely because of Agave jobs. I double-checked yesterday before I made the change.